contracts/PartyBid.sol

```diff
@@ -412,26 +432,108 @@ contract PartyBid is ReentrancyGuardUpgradeable, ERC721HolderUpgradeable {
         _tokens = _value * TOKEN_SCALE;
     }
 
-    // ============ Internal: Bid ============
-
     /**
      * @notice The maximum bid that can be submitted
      * while leaving 5% fee for PartyDAO
      * @return _maxBid the maximum bid
      */
-    function _getMaximumBid() internal view returns (uint256 _maxBid) {
-        _maxBid = (totalContributedToParty * 100) / (100 + FEE_PERCENT);
+    function getMaximumBid() public view returns (uint256 _maxBid) {
+        _maxBid = (totalContributedToParty * 10000) / (10000 + ETH_FEE_BASIS_POINTS);
     }
 
     /**
-     * @notice Calculate 5% fee for PartyDAO
+     * @notice Calculate the amount of fractional NFT tokens owed to the contributor
+     * based on how much ETH they contributed towards the auction,
+     * and the amount of excess ETH owed to the contributor
+     * based on how much ETH they contributed *not* used towards the auction
+     * @param _contributor the address of the contributor
+     * @return _tokenAmount the amount of fractional NFT tokens owed to the contributor
+     * @return _ethAmount the amount of excess ETH owed to the contributor
+     */
+    function getClaimAmounts(address _contributor)
```

Comment:
this is just the same exact implementation as `_calculateTokensAndETHOwed` but moved and renamed, correct?

Reply:
correct, with the added `require` statement so off-chain people don't query this at the wrong time and get confused